.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,19 @@
+name: SwiftLint
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - ".github/workflows/swiftlint.yml"
+      - ".swiftlint.yml"
+      - "**/*.swift"
+
+jobs:
+  SwiftLint:
+    runs-on: macos-latest
```

Comment:
[![imo-badge](https://img.shields.io/badge/review-imo-orange.svg)]()  
Replace the macOS runner with an Ubuntu runner using the SwiftLint Docker image for the following reasons:

- Explicit specification of SwiftLint version.
- Optimization of resource usage (free for public repositories).

```suggestion
    runs-on: ubuntu-latest
    container:
      image: ghcr.io/realm/swiftlint:0.54.0
```